homeassistant/components/enigma2/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the Enigma2 platform."""
+DOMAIN = "enigma2"
+
+CONF_USE_CHANNEL_ICON = "use_channel_icon"
+CONF_DEEP_STANDBY = "deep_standby"
+CONF_SOURCE_BOUQUET = "source_bouquet"
+CONF_MAC_ADDRESS = "mac_address"
+
+ATTR_MEDIA_CURRENTLY_RECORDING = "media_currently_recording"
```

Comment:
Please only move constants that need to be shared between more than one module. Constants that are only used in a single module should be defined in that module.

Reply:
Okay, is done!